### PR TITLE
FRR 7.5

### PR DESCRIPTION
**Pull request by pavel-shirshov**
Don't merge. Just checking.
Build libyang1 which is required for frr 7.5

**- Why I did it**

**- How I did it**

**- How to verify it**

**- Which release branch to backport (provide reason below if selected)**

- [ ] 201811
- [ ] 201911
- [ ] 202006

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by pavel-shirshov** [comment]
retest broadcom please

**Comment by pavel-shirshov** [comment]
retest mellanox please

**Comment by pavel-shirshov** [comment]
retest vs please

**Comment by lguohan** [comment]
looks something wrong with the warm reboot test, can you check?

**Comment by pavel-shirshov** [comment]
retest mellanox please

**Comment by lguohan** [comment]
retest vsimage please

**Comment by lguohan** [comment]
retest mellanox please

**Comment by lguohan** [comment]
retest vsimage please

**Comment by lguohan** [comment]
retest this please

**Comment by lguohan** [comment]
retest vsimage please

**Review by lguohan — APPROVED** [review]

File: rules/libyang1.mk [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+LIBYANG1_CPP = libyang-cpp1_$(LIBYANG1_FULLVERSION)_$(CONFIGURED_ARCH).deb
+$(LIBYANG1_CPP)_DEPENDS += $(LIBYANG1)
+$(eval $(call add_derived_package,$(LIBYANG1),$(LIBYANG1_CPP)))
+
+LIBYANG1_CPP_DEV = libyang-cpp-dev_$(LIBYANG1_FULLVERSION)_$(CONFIGURED_ARCH).deb
+$(eval $(call add_derived_package,$(LIBYANG1),$(LIBYANG1_CPP_DEV)))
+
+LIBYANG1_CPP_DBG = libyang-cpp1-dbgsym_$(LIBYANG1_FULLVERSION)_$(CONFIGURED_ARCH).deb
+$(eval $(call add_derived_package,$(LIBYANG1),$(LIBYANG1_CPP_DBG)))
```
Comment (shi-su) [inline rules/libyang1.mk:31]: The naming of .deb looks a little bit inconsistent. Seems like a mixture of "libyang-*", "libyang1-*", "libyang-*1".

Comment (lguohan) [reply rules/libyang1.mk:31]: this is defined by debian packing name  https://packages.debian.org/source/bullseye/libyang

I think they may have some naming convention, i am not sure.

**Review by shi-su — APPROVED** [review]

**Pull request merged**